xcodeproj/BUILD

```diff
@@ -0,0 +1 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Is this needed?

Reply:
I typically make items in that package public as it is the location to find the publicly available rules, macros, and libraries. I can remove it until/if we need it.